### ISSUE TITLE
Inferred classes should not be editable

### ISSUE THREAD

**Issue by bdionne** [NCIEVS/nci-protege5#340]
* Make a class a subClassOf two disjoint classes.  
* Reasoner will pick up error.  
* Remove one of the disjoint parents.  
* The reasoner still reports a "SubClassOf owl:Nothing" error.  
* Go to class and try to remove owl:Nothing equivalency, get error:

````
Uncaught Exception in thread 'AWT-EventQueue-0'
java.lang.NullPointerException: ontology must not be null
	at org.semanticweb.owlapi.util.OWLAPIPreconditions.checkNotNull(OWLAPIPreconditions.java:98) ~[na:na]
	at org.semanticweb.owlapi.model.OWLOntologyChange.<init>(OWLOntologyChange.java:41) ~[na:na]
	at org.semanticweb.owlapi.model.OWLAxiomChange.<init>(OWLAxiomChange.java:39) ~[na:na]
	at org.semanticweb.owlapi.model.RemoveAxiom.<init>(RemoveAxiom.java:38) ~[na:na]
	at org.protege.editor.owl.ui.frame.AbstractOWLFrameSectionRow.getDeletionChanges(AbstractOWLFrameSectionRow.java:227) ~[na:na]
	at org.protege.editor.owl.ui.framelist.OWLFrameList.handleDelete(OWLFrameList.java:409) ~[na:na]
	at gov.nih.nci.ui.EditPanel$3.handleDelete(EditPanel.java:215) ~[na:na]
	at org.protege.editor.core.ui.list.MList.lambda$new$0(MList.java:41) ~[na:na]
	at org.protege.editor.core.ui.list.MList.handleMouseClick(MList.java:204) ~[na:na]
````

It's ok for the error to still appear, it won't disappear until `Synchronize Reasoner` is used to pull latest results from Pellet.

The bug is that the `owl:Nothing` class that shows up as an equivalent is `inferred` and should not be editable. It should appear without the icons:

<img width="1019" alt="screen shot 2017-11-27 at 5 09 41 pm" src="https://user-images.githubusercontent.com/31741/33292048-d8b103e8-d395-11e7-91c8-404714d4e9b9.png">

**Comment by bdionne** [comment]
Fixed in [V2.1.0.12](https://github.com/bdionne/protege/commit/d9741d7750450614887e70ba7bb8e88d676cc75d)

**Comment by kondareddys2** [comment]
fixed in QA server on V2.1.0.12 dev.Closing the defect.
